src/models/geograph.py

```diff
@@ -167,29 +178,45 @@ def graph(self, new_graph):
         self._graph = new_graph
 
     @property
-    def rtree(self) -> STRtree:
+    def rtree(self) -> rtree_lib.index.Index:
         """Return Rtree object."""
         return self._rtree
 
+    @property
+    def habitats(self) -> Dict[str, nx.Graph]:
+        """Return habitat dictionary."""
+        return self._habitats
+
+    @property
+    def df(self) -> gpd.GeoDataFrame:
+        """Return dataframe containing polygons."""
+        return self._df
+
+    @df.setter
+    def df(self, new_df):
+        self._df = new_df
```

Comment:
Ok, actually found my answer here: 
https://stackoverflow.com/a/36943813

TL;DR: 
It's mainly useful when changing the underlying functionality of how variables are fetched (e.g. by checking against valid options first or so)

Reply:
Since we don't really have any need to do that, I've changed the `graph` and `df` properties to be attributes and the rest can remain properties because we want them to be read-only.